cmd/cloud/server.go

```diff
@@ -130,6 +124,13 @@ func executeServerCmd(flags serverFlags) error {
 	}
 	model.SetUtilityDefaults(flags.utilitiesGitURL)
 
+	if flags.deployLocalMattermostOperator {
+		_, err := os.Stat(helm.LocalMattermostOperatorHelmDir)
+		if err != nil {
+			return errors.Wrapf(err, "failed to ensure directory %s exists; disable server flag deploy-local-mattermost-operator or ensure directory exists", helm.LocalMattermostOperatorHelmDir)
+		}
+	}
```

Comment:
Rather than hard coding the local directory, could you just accept a folder as the argument rather than a boolean? Would allow you to not have to simlink/copy/paste into the helm-charts/mattermost-operator folder